CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### citus-docker v10.2.2.docker (October 14,2021) ###
+
+* Bump Citus version to 10.2.2
```

Comment:
Maybe we should add an entry for alpine base image bump.

```suggestion
* Bump Citus version to 10.2.2

* Bump alpine image PG version to 14.0
```

Reply:
Fixed as you suggested